src/xcode/ENA/ENA/Source/Scenes/Home/HomeViewController.swift

```diff
@@ -151,7 +151,15 @@ final class HomeViewController: UIViewController, RequiresAppDependencies {
 				supportedCountries = []
 			}
 			
-			DispatchQueue.main.asyncAfter(deadline: .now() + 0.5) {
+			var delay = 0.5
```

Comment:
THIS is a perfect place for a comment, why we have this line in here and why it's 1.5 in UI tests!

Reply:
And it is commented 👍

Reply:
Where is this requirement for the delay coming from?

Reply:
This was a requirement in the User Story (Delta Onboarding)